.github/workflows/ui_tests.yml

```diff
@@ -46,19 +46,29 @@ on:
         required: false
 
 jobs:
+  setup_playwright_runners:
+    uses: WalletConnect/gh-actions-runners/.github/workflows/setup-runners.yml@main
+    with:
+      cpu: 16384
+      memory: 65536
+      labels: playwright
+      count: 5
+    secrets:
+      GITHUB_PAT: ${{ secrets.RELEASE_TOKEN }}
+      TF_API_TOKEN: ${{ secrets.TFC_INFRA_TOKEN }}
+
   ui_tests:
     name: 'Playwright Tests'
-    runs-on:
-      group: ubuntu-runners
+    runs-on: [self-hosted, playwright]
     strategy:
       fail-fast: false
       matrix:
         shard: [1, 2, 3, 4, 5]
         shardTotal: [5]
     timeout-minutes: 20
     steps:
-      - name: Tune GitHub-hosted runner network
-        uses: smorimoto/tune-github-hosted-runner-network@v1
+      # - name: Tune GitHub-hosted runner network
+      #   uses: smorimoto/tune-github-hosted-runner-network@v1
```

Comment:
should we remove this?

Reply:
This was an attempt to fix GH flaky networking, but I don't think it ever fixed. The dependency isn't available in the self-hosted runners. We can probably remove either way